configuration/src/keyVaultClient.d.ts

```diff
@@ -0,0 +1,72 @@
+declare module 'azure-keyvault' {
```

Comment:
they don't have typings? 
We used to git force add the typings files we were adding under typings folder.